geostore/migrations/0035_layerextrageom_editable.py

```diff
@@ -1,18 +0,0 @@
-# Generated by Django 2.2.8 on 2019-12-10 10:45
```

Comment:
This migration should not be remove. The migration 0034_auto_20200107_1240.py should be after this one without the 2 operations about layerextrageom